Cleaned up README

agentFinland opened the pull request:
- corrected minor grammar errors mainly where words were missing or in wrong order
- corrected some inconsistant styling
- changed some formatting of "What's great about Steward" section to improve readability
- clarified a few steps in "Getting Started" that had me confused during my first Steward install

agentFinland committed:
Cleaned up README

- corrected minor grammar errors mainly where words were missing or in wrong order
- corrected some inconsistant styling
- changed some formatting of "What's great about Steward" section to improve readability
- clarified a few steps in "Getting Started" that had me confused during my first Steward install

OndraM commented on README.md:
```diff
@@ -7,73 +7,82 @@
+- Simple syntax sugar layer on top of default [WebDriver commands](https://github.com/facebook/php-webdriver/wiki/Example-command-reference) helps shorten your tests and improve readability
+- If you already use PHP, you don't have to learn a new language to write functional tests. Moreover, if you are familiar with unit tests and PHPUnit, you know it all
+- Allows you to plan test dependencies
+   - Ex. need to wait 2 minutes until some event gets through your message queue before testing the result? No problem! The order of tests is optimized to minimize the total execution time
```
Eg? Or if this is shortcut for example, I'd prefer longer version without the shortcut ("For example ..."). Also the same case below.

OndraM commented on README.md:
```diff
@@ -7,73 +7,82 @@
+   - Ex. you can add custom configuration options or change parameters passed to child PHPUnit processes
+- Cloud services like [Sauce Labs](https://saucelabs.com/), [BrowserStack](https://www.browserstack.com/) or [TestingBot](https://testingbot.com/) are fully integrated giving you a chance to run tests with less setup and without your own infrastructure
+- Steward is field tested - we use it daily in [our company](https://www.lmc.eu/english) to maintain the quality of our products thanks to hundreds of test-cases
+   - the library itself is also extensively covered by unit tests
```
I'd keep this as part of the sentence, ie. not as standalone point.

OndraM commented on README.md:
```diff
@@ -7,73 +7,82 @@
+Another possibility is to [start Selenium Server and test browser inside a Docker container][wiki-docker].
 
-#### Get Selenium standalone server
+####To run Selenium locally:
```
This is not properly formatted for markdown. And I think this header is unnecessary - better would IMHO be to reword previous sentences to something like `This step only applies if you want to download and run Selenium Standalone Server with the test browser locally right on your computer. Another possibility is to [start Selenium Server and test browser inside a Docker container][wiki-docker].`.

What do you think, would it be understandable?

agentFinland replied:
I agree with this. I'll make the change.

OndraM reviewed (changes requested):
Hi @agentFinland , this is really a great contribution! This kind of feedback on readability of the readme from newcomers is extremely valuable, so thank you very much for it.

I have few minor points, could you please have a look?

Also - I would prefer keeping periods on end of the sentences (which begins with a capital letter) in bullet point lists. Could you change this, please?

agentFinland committed:
README Edits Suggested by @OndraM

- Changed short forms of example (ex. and e.g.) to full form "for example"
- re-added periods to end of bullet points starting with capital letters
- rephrased start of Step 2 of "Getting Started"
- combined two bullet points as suggested

agentFinland commented:
I agree with the suggested changes and have updated my pull request accordingly. Thanks again @OndraM for the suggestions.
